pkgs/development/python-modules/pygam/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, future
+, numpy
+, pandas
+, progressbar2
+, scipy
+, pytest
+}:
+
+buildPythonPackage rec {
+  pname = "pygam";
+  version = "0.8.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0rli8w0fgfr6mxk5cx3n6i5zafi1j0abl2nslrrdxzmjm2p03bjw";
+  };
+
+  propagatedBuildInputs = [ future numpy pandas progressbar2 scipy ];
+  checkInputs = [ pytest ];
```

Comment:
Note that the check phase doesn't actually succeed in finding/running any tests. To get these working you have to add a custom `checkPhase`:

```
  checkPhase = ''
    cd pygam/tests
    pytest
  '';
```

This passes successfully for me on non-nixos linux x86_64 & macos 10.13.

Reply:
Done!